x-pack/filebeat/module/mysqlenterprise/_meta/config.yml

```
@@ -0,0 +1,10 @@
+- module: mysqlenterprise
+  audit:
+    enabled: true
+
+    # Sets the input type. Currently only supports file
+    #var.input: file
+
+    # Set paths for the log files when file input is used.
+    # Should only be used together with file input
+    # var.paths: /home/user/mysqlauditlogs/audit.*.log
```

I *think* this won't work if the user uncomments the line, as `vars.paths` is expected to be a list not a string.

```suggestion
    # var.paths:
    #  - /home/user/mysqlauditlogs/audit.*.log
```